python/cudf/cudf/tests/test_string.py

```diff
@@ -206,6 +207,43 @@ def test_string_astype(dtype):
     assert_eq(expect, got)
 
 
+def test_string_decimal():
```

Comment:
I believe we could parametrize this test on `data, scale, precision` triplets. Something like:


```python
@pytest.mark.parametrize(
    "data, scale, precision",
    [
        (["1.11", "2.22", "3.33"], 2, 3),
        (["111", "222", "33"], 0, 3),
        ...
    ]
)
def test_string_decimal():
    gs = Series(data)
    fp = gs.astype(cudf.Decimal64Dtype(scale=scale, precision=precision)
    got = fp.astype("str")
    assert(gs, got)
```